src/main/java/de/bonndan/nivio/output/threeD/ThreeDRenderController.java

```diff
@@ -29,7 +29,7 @@
     @Autowired
     public ThreeDRenderController(LandscapeRepository landscapeRepository) throws IOException {
         this.landscapeRepository = landscapeRepository;
-        content = new String(Files.readAllBytes(new File("src/main/resources/static/html/3d.html").toPath()));
+        content = IOUtils.toString(getClass().getResourceAsStream("/static/html/3d.html"), StandardCharsets.UTF_8);
```

Comment:
:+1: for using resourceAsStream

However, I'd prefer not to introduce a new dependency for resource loading. There might be something from Spring or Java ready to use.